airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/utils/connector_runner.py

```diff
@@ -44,37 +44,38 @@ def output_folder(self) -> Path:
     def input_folder(self) -> Path:
         return self._volume_base / f"run_{self._runs}" / "input"
 
-    def _prepare_volumes(self, config: Optional[Mapping], state: Optional[Mapping], catalog: Optional[ConfiguredAirbyteCatalog]):
+    def _prepare_image(self, config: Optional[Mapping], state: Optional[Mapping], catalog: Optional[ConfiguredAirbyteCatalog]):
```

Comment:
This is a patch whose reason is detailed [here](https://github.com/airbytehq/airbyte/blob/e388dbf5205d3f86770edb868b38cc4680a7a34a/tools/ci_connector_ops/ci_connector_ops/pipelines/actions/tests.py#L119) and that I'd prefer to avoid.